pkg/metadata/sandbox.go

```diff
@@ -150,3 +164,115 @@ func (s *sandboxStore) List() ([]*SandboxMetadata, error) {
 func (s *sandboxStore) Delete(sandboxID string) error {
 	return s.store.Delete(sandboxID)
 }
+
+type SandboxNetNS struct {
+	sync.Mutex
+	NS       ns.NetNS
+	Symlink  *os.File
+	Closed   bool
+	Restored bool
+}
+
+func (ns *SandboxNetNS) symlinkCreate(name string) error {
```

Comment:
Since I referenced the cri-o v0.3, so I may write something outdated.
So shell I reference the newest version?